qiskit/qpy/binary_io/schedules.py

```diff
@@ -95,6 +94,7 @@ def _loads_symbolic_expr(expr_bytes, use_symengine=False):
         return common.load_symengine_payload(expr_bytes)
     else:
         from sympy import parse_expr
+        import symengine as sym
```

Comment:
Do we need to add a check for the optional install of symengine here (or at a higher level)? https://github.com/Qiskit/qiskit/blob/a18e15163d8a93c0e072d4d3c50151a2cd9f036a/qiskit/utils/optionals.py#L333

Reply:
It should already happen at a higher level since this is pretty deep in the call stack, but I can add the decorator here too just out an abundance of caution it doesn't really hurt.

Reply:
Actually looking at this again this whole function isn't necessary anymore. Since we don't support pulse all that matters is that we read the appropriate length for old versions that support it. We can't generate a pulse schedule anymore so there is no advantage to actually parsing the data, since we'll just throw it away as not representable in a circuit. I'll just have this always return `None` and we don't need to import it at all.